content/en/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Autoscaling Workloads
+description: >-
+  With autoscaling, you can automatically update your workloads in one way or another. This allows your cluster to react to changes in resource demand more elastically and efficiently.
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+In Kubernetes, you can _scale_ a workload depending on the current demand of resources.
+This allows your cluster to react to changes in resource demand more elastically and efficiently.
+
+When you scale a workload, you can either increase or decrease the number of replicas managed by
+the workload, or adjust the resources available to the replicas in-place.
+
+The first approach is referred to as _horizontal scaling_, while the second is referred to as
+_vertical scaling_.
+
+There are manual and automatic ways to scale your workloads, depending on your use case.
+
+<!-- body -->
+
+## Scaling workloads manually
+
+Kubernetes supports _manual scaling_ of workloads. Horizontal scaling can be done
+using the `kubectl` CLI.
+For vertical scaling, you need to _patch_ the resource definition of your workload.
```

Comment:
should there be a note about VPA ? It's an extra install so I would say no but it is part of k/autoscaler
https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler

Reply:
If I don't get you wrong, there is a whole paragraph about the VPA:

<img width="706" alt="Screenshot 2024-02-16 at 17 26 51" src="https://github.com/kubernetes/website/assets/28064429/03705bec-340f-4352-9ce1-1aa21785905e">

Or are you referring to something else entirely?

Reply:
https://k8s.io/docs/concepts/workloads/autoscaling/#scaling-workloads-vertically